rskj-core/src/main/java/co/rsk/mine/gas/StableMinGasPriceProviderFactory.java

```diff
@@ -0,0 +1,43 @@
+package co.rsk.mine.gas;
+
+import co.rsk.config.mining.HttpGetStableMinGasSystemConfig;
+import co.rsk.config.mining.StableMinGasPriceSystemConfig;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+
+public class StableMinGasPriceProviderFactory {
```

Comment:
I have a few comments on the factory, but I really liked how you approach the problem. This factory would be exactly one of the things that crossed my mind to apply when I saw the ticket. Good job!